main/inc/lib/main_api.lib.php

```diff
@@ -2560,6 +2562,16 @@ function api_is_teacher() {
 }
 
 /**
+ * Checks whether the current user is a invited user
+ * @return boolean
+ */
```

Comment:
In main_api.lib.php all functions should have this name convention "api_is_invited_user" for legacy reasons.